search-domain/build.gradle

```diff
@@ -1,3 +1,46 @@
+buildscript {
```

Comment:
I didn' change anything about the structure of the build scripts, just changed the versions and minimal number of dependencies. These `buildscript {...}` additions appear to be a github or git merge fluke.